src/bip85.rs

```diff
@@ -0,0 +1,49 @@
+use bdk::bitcoin::{
+    hashes::{sha512, Hash, HashEngine, Hmac, HmacEngine},
+    secp256k1::Secp256k1,
+    util::bip32::{ChildNumber, DerivationPath, ExtendedPrivKey},
+};
+
+pub fn get_bip85_bytes<const L: usize>(
+    xpriv: ExtendedPrivKey,
+    app_num: u32,
+    index: u32,
+) -> [u8; L] {
+    let path = DerivationPath::from(vec![
+        ChildNumber::Hardened { index: 83696968 },
+        ChildNumber::Hardened { index: app_num },
+        ChildNumber::Hardened { index: index },
```

Comment:
according to the [BIP](https://github.com/bitcoin/bips/blob/master/bip-0085.mediawiki#hex) `L` is meant to be in the derivation path. It should be:

```
m/83696968'/128169'/{num_bytes}'/{index}'
```